client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -20,21 +91,41 @@ const TaskFormPage = () => {
             header={"Task Title"}
             type={"text"}
             placeholder={"Task Name"}
+            onChange={(e) => updateTaskFormData(e, "title")}
+            value={taskFormData.title}
           />
           <InputBox
             header={"Location"}
             type={"text"}
             placeholder={"Location Name"}
+            onChange={(e) => updateTaskFormData(e, "location")}
+            value={taskFormData.location}
           />
           <InputBox
             header={"Description"}
             type={"text"}
             placeholder={""}
             expanded={true}
+            onChange={(e) => updateTaskFormData(e, "description")}
+            value={taskFormData.description}
+          />
+          <p className={styles.inputHeader}>Start</p>
+          <DateSelector
+            showTime={true}
+            selectedDate={new Date(taskFormData.startDate)}
```

Comment:
Referring to one of the comments on line 19, since we'll no longer have the helper function that you created nor the .toISOString method, we won't need `new Date(taskFormData.startDate)`. Rather we can just have taskFormData.startDate since it'll already be of type date.

Reply:
Changed to just `taskFormData.startDate` since the date object is stored in the state now.